src/quill-cursors/quill-cursors.ts

```diff
@@ -110,6 +111,16 @@ export default class QuillCursors {
   private _registerDomListeners(): void {
     const editor = this.quill.container.getElementsByClassName('ql-editor')[0];
     editor.addEventListener('scroll', () => this.update());
+    editor.addEventListener('touchstart', this._handleCursorTouch);
+    editor.addEventListener('click', this._handleCursorTouch);
```

Comment:
Why are we handling `click`?

Reply:
We shouldn't handle it, it's missed from the previous implementations. I removed